bids/layout/layout.py

```diff
@@ -1404,7 +1404,7 @@ def get_tr(self, derivatives=False, **filters):
         return all_trs.pop()
 
     def build_path(self, source, path_patterns=None, strict=False,
-                   scope='all', validate=True):
+                   scope='all', validate=True, absolute_paths=None):
```

Comment:
Please roll the change on permissions back to 644 on this file.

Reply:
Whoops! Fixed.